superset/templates/superset/dashboard.html

```diff
@@ -22,4 +22,5 @@
   <div id="grid-container" class="slice-grid gridster"></div>
 
 </div>
+<input type="hidden" name="csrf_token" id="csrf_token" value="{{ csrf_token() }}">
```

Comment:
I had to add it since it redefines the body block. Eventually won't need it when we reactify the dashboard.